README.rst

```diff
@@ -49,3 +49,21 @@ Repository Overview
 .. _pypyjs-examples: https://github.com/pypyjs/pypyjs-examples
 .. _pypyjs.github.io: https://github.com/pypyjs/pypyjs.github.io
 .. _pypyjs.org: https://pypyjs.org
+
+Transpiling es6
+~~~~~~~~~~~~~~~
```

Comment:
Can you please move this description into CONTRIBUTING.rst?  I think it'll be a better fit there than in the main README.